src/Lykke.Service.Subscribers/Modules/ServiceModule.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Autofac;
+using AzureStorage.Tables;
+using Common.Log;
+using Lykke.Service.Subscribers.AzureRepositories.Subscribers;
+using Lykke.Service.Subscribers.Core.Repositories;
+using Lykke.Service.Subscribers.Core.Services;
+using Lykke.Service.Subscribers.Core.Settings.ServiceSettings;
+using Lykke.Service.Subscribers.Services;
+using Lykke.SettingsReader;
+
+namespace Lykke.Service.Subscribers.Modules
+{
+    public class ServiceModule : Module
+    {
+        private readonly IReloadingManager<SubscribersSettings> _settings;
+        private readonly ILog _log;
+
+        public ServiceModule(IReloadingManager<SubscribersSettings> settings, ILog log)
+        {
+            _settings = settings;
+            _log = log;
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder.RegisterInstance<ISubscriberRepository>(
```

Comment:
Register ISubscriberRepository via RegisterType, but AzureTableStorage via RegisterInstance

Reply:
Is there a problem if it is this way?